src/components/dev-dot-content/dev-dot-content.module.css

```diff
@@ -20,6 +20,7 @@
       border-radius: 6px;
       color: var(--token-color-palette-neutral-600);
       padding: 16px;
+      margin-bottom: 20px;
```

Comment:
I'm not sure what the recommended vertical rhythm is for these docs components? I chose `20px` to match the bottom margin of the table component below but please lmk if that value should be different.

Reply:
So far, we've mostly added `margin-top` to elements inside of docs content. Brayden and I worked together closely on spacing between different kinds of content awhile ago, so you could check this value with him if you feel it's necessary!

In this case since we're already setting `margin-bottom` on the table component, I'd probably recommend setting the `margin-top` there as well so the relationship in spacing values is clearer.

Reply:
(crossed wires there, +1 on Ashlee's take on setting both margin-top and margin-bottom. Once we have a more unified system, I think we'll be able to pare it back to margin-top only if we really want, though setting both gives us some nice flexibility due to margin-collapse which may come in handy in meeting design spacing needs).

Reply:
> Once we have a more unified system, I think we'll be able to pare it back to margin-top only if we really want

+1!

Also, @zchsh and I are always commenting at almost the same time. 🤣

Reply:
We initially were using 16px but 20px is also pretty nice. Let's not over engineer this and move forward, can refine later as needed.

Reply:
@ashleemboyer @zchsh sorry I saw these after I put my comment in! Let's work on a more unified system for how we handle margins (post June 20th) :)

Reply:
Awesome! i'll set `margin-top` to 20px ⛵